lib/features/giving_flow/models/organisation.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:equatable/equatable.dart';
+
+class Organisation extends Equatable {
+  const Organisation({
+    required this.collectGroupId,
+    required this.name,
+    required this.goal,
+    this.logoLink,
+    this.thankYou,
+  });
+
+  const Organisation.empty()
+      : this(
+            collectGroupId: '',
+            name: 'Mock Organisation Long Name',
```

Comment:
Will the user see these texts? And is that discussed with Andy?

Reply:
No the user wont see this